### PR TITLE
JIRA ID-5901: Blueprint Table Component - Primary Link

### PR DESCRIPTION
**Pull request by BivashEpsi**
Below points are covered as part of this pull request:
1. we can have 1 link per table row
2. The link can be in any column
3. The link should have a valid href attribute value
4. The link uses .link-primary
5. We can add an ARIA label for text that is not unique

### CODE REVIEW

File: src/app/table/table.component.html [hunk 1]
```diff
@@ -33,6 +33,15 @@
         </tbody>
         <ng-template #loaded>
           <tbody *ngIf="tableData.length > 0; else noColumnData">
+            <tr *ngFor="let dataList of tableData; ">
```
Comment (kaseyllabelle) [inline src/app/table/table.component.html]: We're already rendering a table row for each record on line 45, so we shouldn't create an additional row for each record here. This is also missing a closing tag (`</tr>`).

File: src/app/table/table.component.html [hunk 2]
```diff
@@ -33,6 +33,15 @@
...
+              <td>{{ dataList.date }}</td>
```
Comment (kaseyllabelle) [inline src/app/table/table.component.html]: The data in the JSON file is placeholder data. We shouldn't hardcode anything based on that data.

File: src/app/table/table.component.html [hunk 3]
```diff
@@ -33,6 +33,15 @@
...
+              <td>{{ dataList.amount }}</td>
+              <td>
+                <a class =".link-primary" attr.aria-label="{{dataList?.ariaLabel}}" href={{dataList.url}}>
```
Comment (kaseyllabelle) [inline src/app/table/table.component.html]: Classes are only referenced with a . in CSS files. In HTML files, the syntax is `class="link-primary"`.

There's also an extra space after the class attribute / before the =. Please format your code before creating a PR so white space and formatting remains clean and consistent.

**Comment by BivashEpsi** [comment]
This pull request Was created from a fork repository, As we are following the Branching System. Hence I have Closed this Pull Request and created a new Pull Request.
Please find below the Link For the new Pull Request:
https://github.com/epsilon-ux/blueprint-table/pull/33

I have also fixed the Review Points provided by Kasey in the new Pull Request.

**Pull request closed without merging**